.github/workflows/stackhpc-all-in-one.yml

```
@@ -22,7 +22,7 @@ on:
       vm_image:
         description: Image for the all-in-one VM
         type: string
-        default: CentOS-stream8
+        default: 7efda0b1-e296-447f-896e-c066d0af5c53
```

```suggestion
        default: bb8c0a34-533f-42fb-a49b-3461e677f3f6
```
Copied from the wrong job. Using this one: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4450001402/jobs/7815037603#step:8:56